README.md

```diff
@@ -0,0 +1,21 @@
+<!--
+  SPDX-FileCopyrightText: 2024 Junde Yhi <junde@yhi.moe>
+  SPDX-License-Identifier: CC0-1.0
+-->
+
+# GNAT Ada Compiler as Freedesktop SDK Extension
+
+This folder contains necessary files to repack an GNAT-based Ada development environment into an extention to the `org.freedesktop.Sdk` Flatpak container.
+
+## Includes
+
+- GNAT-FSF (Ada/GCC)
+- GNATprove (SPARK)
+- GPRbuild
+- Alire (`alr`)
+
+So far the manifest simply downloads pre-built binaries from [alire-project/GNAT-FSF-builds](https://github.com/alire-project/GNAT-FSF-builds/releases).
```

Comment:
Maybe we should add in the README that it is x86_64 for now. But all else looks good to me :)